systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -227,6 +231,14 @@ function checkDirty(obj)
              all(u==obj.LCP_cache.data.u) && num_args_out <= obj.LCP_cache.data.nargout);
     end
 
+    function [z, Mqdn, wqdn] = solveLCP_mex(obj, t, x, u)
```

Comment:
so far function names that end with mex have only been used for actual mex functions. Do we want to stick to that convention?
Also, we shouldn't have underscores in method names.